### ISSUE TITLE
Normalization when combining SI and FA

### ISSUE THREAD

**Issue by OMBengtsson** [brianstock/MixSIAR#288]
Hi!
I have been looking into the possibility to use MixSIAR for a combined dataset with both stable isotopes and fatty acids as tracers, as done in the study by [O'Donovan et al. 2018](https://doi.org/10.1002/ecs2.2420). 
The following quote is from their paper and addresses the problem of combining continuous (SI) and compositional (FA) data:  "Importantly, the two biomarkers cannot be merged and used in the Bayesian mixing model without a transformation to put them on the same scale of measurement. Accordingly, the SI–FA dataset was transformed by subtracting the mean and dividing by the standard deviation (Dethier et al. 2013), making the two biomarkers quantitatively comparable during modeling."
My question is then - doesn't the run_model function already normalize the tracer data before modelling - making any prior normalization redundant? 
Best regards
/Olof

**Comment by brianstock-NOAA** [comment]
Yes, you are correct, `run_model` normalizes the tracer data so they do not need to be normalized beforehand. For details, see [lines 211-254](https://github.com/brianstock/MixSIAR/blob/master/R/run_model.R#L211) and comment on [30-38](https://github.com/brianstock/MixSIAR/blob/master/R/run_model.R#L30).
